### PR TITLE
started learning backend development

### PR DESCRIPTION
**Pull request by rajsinghkog**

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from email.policy import default
+from sqlalchemy import Boolean, Column, Integer, String
+from owndatabase import Base
+
+
+
+class ContactList(Base):
+    __tablename__ = "contactlist5"
```
Comment (adityaiitg) [inline model.py]: Try to have a better table name.

**Review by adityaiitg — COMMENTED** [review]
Please address the issues

File: model.py (hunk 1 above)
Comment (rajsinghkog) [reply model.py]: let me correct this.

Comment (rajsinghkog) [reply model.py]: done

**Pull request merged**